config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/jinzhu/configor"
+)
+
+//FromJSON will load the requested JSON config into the given config interface
+func FromJSON(config interface{}, name string) error {
+
+	// we need to get the absolute path and also test it because of the
+	// difference of how this code runs in different environments
+	const configPath = "./_config/"
+	absPath, _ := filepath.Abs(configPath + name)
```

Comment:
We should check and fail on all errors.  Also, we should use `filepath.Join` to join paths.  So, `filepath.Abs(filepath.Join("_config", name))`.